internal/cloud/update_uploader.go

```diff
@@ -117,16 +140,42 @@ func (u *UpdateUploader) makeUpdates(st store.RStore) []update {
 	u.lastFinishTime = highWaterMark
 	u.lastCompletedBuildCount = state.CompletedBuildCount
 
-	return result
+	return updateSet{
+		TeamID:  teamID{ID: state.TeamName},
+		Updates: updates,
+	}
 }
 
-func (u *UpdateUploader) sendUpdates(ctx context.Context, updates []update) {
+func (u *UpdateUploader) sendUpdates(ctx context.Context, updates updateSet) {
+	buf := bytes.NewBuffer(nil)
+	encoder := json.NewEncoder(buf)
+	err := encoder.Encode(updates)
+	if err != nil {
+		logger.Get(ctx).Infof("Error encoding updates: %v", err)
+		return
+	}
+
+	request, err := http.NewRequest(http.MethodPost, u.putUpdatesURL(), buf)
+	if err != nil {
+		logger.Get(ctx).Infof("Error sending updates: %v", err)
+		return
+	}
+
```

Comment:
we want to set the token header here so that the server can get the user id

Reply:
done!